docker-for-ibm-cloud/administering-swarms.md

```diff
@@ -0,0 +1,266 @@
+
+### Download client certificates
+[Download the client certificate bundle](/datacenter/ucp/2.2/guides/user/access-ucp/cli-based-access/#download-client-certificates) to create objects and deploy services from a local Docker client.
+
+1. [Access UCP](#access-ucp).
+2. Under your user name (for example, *admin*), click *My Profile*.
+3. Click *Client Bundles* > *New Client Bundle*. A zip file is generated.
+4. In the GUI, you are now shown a label and public key. You can edit the label by clicking the pencil icon and giving it a name, e.g., _d4ic-ucp_.
+5. In a terminal, navigate and unzip the client bundle.
+
+   ```bash
+   $ cd Downloads && unzip ucp-bundle-admin.zip
+   ```
+
+   > Keep your client bundle handy
+   >
+   > Move the certificate environment variable directory to a safe and accessible location on your machine. You'll use it a lot!
+
+6. From the client bundle directory, update your `DOCKER_HOST` and `DOCKER_CERT_PATH` environment variables by running the `env.sh` script.
+   ```bash
+   $ source env.sh
+   ```
+
+   > Set your environment to use Docker EE for IBM Cloud
+   >
+   > Repeat this to set your environment variables each time you enter a new terminal session, or after you unset your variables, to connect to the Docker EE for IBM Cloud swarm.
+
+7. Verify that your certificates are being sent to Docker Engine. The command returns information on your swarm.
+   ```bash
+   $ docker info
+   ```
+
+## View swarm resources
+
+### Cluster-level resources
+To review resources used within a particular Docker EE cluster, use the CLI or UCP.
+
+**CLI**: The `bx d4ic` CLI lists, modifies, and automates cluster infrastructure, as well as the initial information to log in to UCP.
+  * Use `bx d4ic list --sl-user user.name.1234567 --sl-api-key api_key` to review a list of your clusters and their UCP URLs.
+  * Use `bx d4ic show --swarm-name my_swarm --sl-user user.name.1234567 --sl-api-key api_key` to review details about the cluster, such as the IP address of manager nodes or the status of the cluster load balancers.
+
+**UCP**: The Docker EE Universal Control Plane provides a web-based GUI to manage swarm users and deployed applications. You can view swarm-related stacks, services, containers, images, nodes, networks, volumes, and secrets.
+
+### Account-level resources
+For an account-level view of services and infrastructure that can be used in your swarm, log in to your [IBM Cloud](https://console.bluemix.net/) account.
+* The IBM Cloud dashboard provides information on connected IBM Cloud services in the account, such as Watson and Internet of Things.
+* The IBM Cloud infrastructure portal shows account infrastructure resources such as virtual devices, storage, and networking.
+
+### Other resources
+For logging and metric data from your swarm, you must first [enable logging for the cluster](logging.md), and then access the data in your IBM Cloud organization and space.
```

Comment:
Maybe "To gather logging and metric data from your swarm,..."

And again, I don't like "you must"